### PR TITLE
fix: form look and layout, mobile layout shift and arrow of slider component and video date style 

### PR DESCRIPTION
**Pull request by VedP20**
What does this PR do?

Fix: form look and layout, mobile layout shift and arrow of slider component and video date style 

Fixes #129
Fixes #35
Screenshot 2023-06-10 193009
Screenshot 2023-06-10 192849
Screenshot 2023-06-10 192910
Screenshot 2023-06-10 192952

Type of change

- New feature (non-breaking change which adds functionality)

## Mandatory Tasks

- [X] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Comment by VedP20** [comment]
Kindly review this PR and let me know if any thing need to be changed?

File: components/UI/Contact.jsx [hunk 1]
```diff
@@ -15,26 +15,26 @@ const Contact = () => {
       name: event.target.name.value,
       email: event.target.email.value,
       message: event.target.message.value,
-    }
-    console.log(data)
+    };
+    console.log(data);
```
Comment (piyushgarg-dev) [inline components/UI/Contact.jsx:19]: Remove console.log

File: components/UI/Contact.jsx [hunk 2]
```diff
@@ -15,26 +15,26 @@ const Contact = () => {
...
     try {
       const response = await axios.post("/api/contact", data);
       if (response.status === 200) {
-        setSubmitted(true)
-        console.log("Form submitted")
+        setSubmitted(true);
+        console.log("Form submitted");
```
Comment (piyushgarg-dev) [inline components/UI/Contact.jsx:24]: Remove console.log

File: components/UI/Contact.jsx [hunk 3]
```diff
@@ -15,26 +15,26 @@ const Contact = () => {
...
       } else {
-        console.log("Failed")
+        console.log("Failed");
```
Comment (piyushgarg-dev) [inline components/UI/Contact.jsx:26]: Show some toast to the user

**Review by piyushgarg-dev — CHANGES_REQUESTED** [review]
Hi, Please check the mentioned changes

**Pull request merged**